pymc_marketing/mmm/transformers.py

```diff
@@ -18,6 +25,8 @@ def batched_convolution(x, w, axis: int = 0):
         to use in the convolution.
     axis : int
         The axis of ``x`` along witch to apply the convolution
+    mode : (ConvMode, optional): The mode of the convolution. Determines how the convolution is applied at the boundaries of x. Defaults to ConvMode.Before.
```

Comment:
Could you please expand on the cases where a specific mode might be needed?

like 
* After: "Adstock" effect
* Before: "Excitement", or "Last minute" effect
* Overlap: "Pull-Forward" and "Pull-Backward" effects